server/models/answer.js

```diff
@@ -0,0 +1,22 @@
+const mongoose = require('mongoose');
+
+const answerSchema = mongoose.Schema(
+  {
+    question_id: { type: mongoose.Types.ObjectId, required: true },
+    body: String,
+    votes: Number,
+    author: {
+      _id: mongoose.Types.ObjectId,
```

Comment:
Same as above, only `mongoose.Schema.Types.ObjectId` works for me. [ObjectIDs](http://mongoosejs.com/docs/schematypes.html#objectids)